content/en/docs/security/secrets-management.md

```diff
@@ -0,0 +1,276 @@
+having a single secrets that has all the access, versus having too many secrets that are
+always used in combination.
+
+Sharing the same secret across different scopes, just because they have the same permissions
+may lead to disruption if such secret needs to be quickly rotated. 
+
+### Disaster Recovery
+
+The entire provisioning of your infrastructure and application must take into account
+break the glass procedures that are secure, provide relevant security controls (e.g. auditing)
+and cannot be misused to bypass other processes (e.g. Access Management).
+
+Around disaster recovery scenarios, consider how they align with your Availability and
+Confidentiality requirements.
+
+### Don't co-locate cipher-text with encryption keys
+
+It should go without saying, but never place secrets together that can provide privilege
+escalation routes. For example, if you store the decryption key for your secrets in GitHub secrets,
+and all your encrypted secrets are stored in the same repository, a single GitHub account
+(with enough access) compromised is enough for all your secrets to be decrypted.
+
+Instead, segregate encryption keys from cypher-text and understand what needs to be compromised
+for the data to be at risk.
+
+### Single Points of Failure
+
+Identify all potential single points of failure and ensure that there is a way around them.
+If all your secrets are encrypted using an encryption key stored in Vault, and due to a major
+failure your Vault instance is completely lost, and no backup is to be found, the encrypted
+secrets are now useless. Therefore, think big picture, and ensure that each step of the way
+has a redundancy and that process is regularly exercised.
+
+The same goes for temporary single points of failure. If you rely on a Key Hierarchy Architecture
+based on a cloud KMS to provision an on-premises cluster/application, consider the impact
+they would have in case of a failure pre, mid or post deploy (of either cluster or applications).
+
+### Ephemeral or Single-use Secrets
+
+The easiest type of secrets to manage are the ones that ephemeral, context- and time-bound.
+They are not supported by all use-cases, however, whenever they are, prioritise them over
+static or long-lived secrets.
+
+An example of Ephemeral secret that is time-bound is tokens provided by cloud providers to
+any application running withing a given Cloud Machine. Those tokens are generated automatically,
+and have a short expiration time. In some cases you can even tie them to a network boundary, 
+meaning that even if they get breached, they won't be able to be used outside the current
+context. 
+
+Flux supports [contextual authorization] for the major Cloud Providers, be aware of the supported
+features and use them whenever possible.
+
+### Detect "chicken and egg" scenarios
+
+Flux won't protect yourself from yourself. On a running cluster, it is quite easy to incrementally
```

Comment:
"won't protect you from yourself" this maybe reads more naturally (though both are acceptable!)